bin/check_pylint_diff

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env bash
+# Copyright (C) 2016 Google Inc., authors, and contributors <see AUTHORS file>
+# Licensed under http://www.apache.org/licenses/LICENSE-2.0 <see LICENSE file>
+# Created By: miha@reciprocitylabs.com
+# Maintained By: miha@reciprocitylabs.com
+
+set -o nounset
+set -o errexit
+
+ARG1=${1:-}
+TMP_REPO=".temp_repo"
+SCRIPT=$(basename "$0")
+SCRIPTPATH=$( cd "$(dirname "$0")" ; pwd -P )
+
+cd "${SCRIPTPATH}/.."
+
+
+if [[ "$ARG1" == "-h" || "$ARG1" == "--help" ]]; then
+  echo "
+Usage: $SCRIPT [test_commit] [-h]
+
+This script will compare pylint error count from two different commits.
+Note: all changes that are not committed will be ignored.
+
+
+Given the commit tree:
+
+       D---E---F---G---H
+            \\         /
+             A---B---C
+
+- Running '$SCRIPT' on C will check the diff between B and C
```

Comment:
>  Imagine you're on master branch with only commits A B C

Does the figure above convey this? Aren't branches usually spun _off_ of master? I might be misinterpreting the figure.

I feel like the script should compare the pull-request HEAD's results to that of:
```bash
git remote add upstream gh:biolab/orange3
git fetch upstream master
THIS_COMMIT=$(git merge-base HEAD upstream/master)
```
I really don't understand how this could not work. All topic branches are (should be) spun off (and rebased on) upstream/master. But in the alternative case, it should be possible to do it explicitly: checkout pull request, evaluate, checkout upstream/master (biolab/orange3/master), evaluate, compare?

I don't see how comparing just the last two commits can be useful at all.